dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.xmlWSClientSecurity-3.0.feature

```diff
@@ -1,12 +1,12 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=io.openliberty.xmlWSClient.3.0-appSecurity.4.0
+symbolicName=io.openliberty.xmlWSClientSecurity-3.0
 visibility=private
 IBM-App-ForceRestart: uninstall, \
  install
 IBM-Process-Types: client
-IBM-Provision-Capability:  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.xmlWSClient-3.0))",\
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.appSecurity-4.0))"
+IBM-Provision-Capability:  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.xmlWSClient-3.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.appSecurityClient-1.0)(osgi.identity=io.openliberty.appSecurity-4.0)))"
```

Comment:
Can we just update com.ibm.websphere.appserver.jaxwsClientSecurity-2.2 to have an OR for jaxwsClient-2.2, jaxwsClient-2.3 and xmlWSClient-3.0 and not have to have a new auto feature at all?